plugins/grpc/kotlin/v1.3.0/buf.plugin.yaml

```diff
@@ -4,8 +4,8 @@ plugin_version: v1.3.0
 source_url: https://github.com/grpc/grpc-kotlin
 description: Generates Kotlin client and server stubs for the gRPC framework.
 deps:
-  - plugin: buf.build/grpc/java:v1.54.0
-  - plugin: buf.build/protocolbuffers/kotlin:v22.2
+  - plugin: buf.build/grpc/java:v1.54.1
```

Comment:
Why isn't grpc/java latest version `v1.55.1 and protocolbuffers/kotlin version `v23.1` ?

There's probably a reason, should we add a comment for those not aware?

Reply:
For now - just doing incremental changes to the levels of other plugins (like connect-kotlin), but if this plugin is ever updated, we can definitely bump further.

Reply:
Yep, sounds perfectly reasonable!